src/WeatherApi/WeatherApi.csproj

```diff
@@ -11,8 +11,6 @@
   <!-- Add support for Ahead-of-Time (AoT) compilation -->
   <PropertyGroup Condition="!$([System.OperatingSystem]::IsMacOS())">
```

Comment:
macOS is supported in NativeAOT in .NET 8. So this condition can be removed if you want.

Reply:
So it works here, but it _doesn't_ work in Polly 🤔 ([logs](https://github.com/App-vNext/Polly/actions/runs/6880981184/job/18716366049?pr=1800#step:5:44))

```
Error: /Users/runner/.dotnet/sdk/8.0.100/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.FrameworkRefe enceResolution.targets(90,5): error NETSDK1204: Ahead-of-time compilation is not supported on the current platform 'osx-x64'. [/Users/runner/work/Polly/Polly/test/Polly.AotTest/Polly.AotTest.csproj]
```

Looks like it's during `dotnet clean` (which this repo doesn't do) so maybe that's a bug somewhere?

Reply:
FYI: https://github.com/dotnet/sdk/issues/36960